src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpApplicationPreambleControllerTests.cs

```diff
@@ -1810,10 +1810,7 @@ public void Provider_changing_route_to_employer_is_directed_to_conditions_of_acc
 
             var redirectResult = result as RedirectToActionResult;
             redirectResult.Should().NotBeNull();
-            redirectResult.ActionName.Should().Be("ConditionsOfAcceptance");
-            redirectResult.ControllerName.Should().BeNull();
-
-            _qnaApiClient.VerifyAll();
+            redirectResult.ActionName.Should().Be("UpdateApplicationProviderRoute");
```

Comment:
How come the redirection has changed? Is this change in behaviour?

Reply:
when implemented PRG, "UpdateApplicationProviderRoute" is the first redirection